lib/window.js

```diff
@@ -60,15 +62,16 @@ class DisplayService extends EventEmitter2 {
       height: _this.opts.height,
       x: _this.opts.x,
       y: _this.opts.y,
+      hasShadow: false,
```

Comment:
we have Boolean like kiosk without the has / is but now we have hasShadow. Should we be consistent